src/System.Console/src/System.Console.csproj

```diff
@@ -159,7 +170,7 @@
     </Compile>
   </ItemGroup>
   <!-- Unix -->
-  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' And '$(TargetGroup)' != 'net46'">
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' And '$(TargetGroup)' != 'net46' And '$(TargetGroup)' != 'net463'">
```

Comment:
Looks like we should make this condition match the other one. `'$(TargetGroup)' == '' OR '$(TargetGroup)'=='netstandard1.3'` I much prefer the inclusive vs exclusive on these TargetGroup conditions.